appdata.patch

```diff
@@ -0,0 +1,37 @@
+--- a/utilities/texmaker.appdata.xml
++++ b/utilities/texmaker.appdata.xml
+@@ -21,22 +21,23 @@
+  </description>
+ <screenshots>
+   <screenshot type="default">
+-    <image height="758" width="1280">https://www.xm1math.net/texmaker/assets/img/texmaker_top_linux.png</image>
```

Comment:
This patch should go upstream if possible

Reply:
Already sent